spec/schemas/security._common.yaml

```
@@ -307,7 +307,7 @@ components:
       properties:
         config:
           type: array
-          description: List of configs to be upgraded.
+          description: A list of configurations to be upgrade.
```

Updated to active voice.

```suggestion
          description: A list of configurations to upgrade.
```